x-pack/plugins/cases/public/components/all_cases/selector_modal/use_cases_add_to_existing_case_modal.tsx

```diff
@@ -19,12 +19,10 @@ import { useCreateAttachments } from '../../../containers/use_create_attachments
 type AddToExistingFlyoutProps = AllCasesSelectorModalProps & {
   toastTitle?: string;
   toastContent?: string;
-  attachments?: CaseAttachments;
 };
 
-export const useCasesAddToExistingCaseModal = (props: AddToExistingFlyoutProps) => {
+export const useCasesAddToExistingCaseModal = (props: AddToExistingFlyoutProps = {}) => {
```

Comment:
Let's change the type of `props` to`props: AddToExistingFlyoutProps | undefined = {}` so consumers of the hook will not need to pass an empty object. For example, `useCasesAddToExistingCaseModal({});`

Reply:
`(props: AddToExistingFlyoutProps = {})` the props will default to an empty object I fixed other parts of the code that were calling this hook with an empty object.

Reply:
You are right. My mistake.